jsk_topic_tools/src/lightweight_throttle_nodelet.cpp

```diff
@@ -94,13 +105,21 @@ namespace jsk_topic_tools
       advertised_ = true;
       pub_ = pnh_.advertise(opts);
     }
+
     ros::Time now = ros::Time::now();
-    if ((now - latest_stamp_).toSec() > 1 / update_rate_) {
+
+    if (latest_stamp_ > now) {
+      ROS_WARN("Detected jump back in time.");
```

Comment:
ok, let's split this PR into `dynamic change update_rate` and `jump back in time`
and question is

1. when jump back in time occurs, only is simulation or does real robot will set back time time?
2. when we need lightweight throttle, other than DRC? can we run all node within in the robot network, or do we have better wifi connection?